creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -109,7 +109,7 @@ const healthCheckController = async (req) => {
 
   const { stateMachineQueueLatestJobSuccess } = response
   if (enforceStateMachineQueueHealth && stateMachineQueueLatestJobSuccess) {
-    const healthyThresholdMs = 5 * config.get('snapbackJobInterval')
+    const healthyThresholdMs = 1000 * 60 * 60 // 1 hour
```

Comment:
lets define this const at top of file at least, if not in config.js (if we want to set diff values for local, stage, prod - will need it in as envvar)

Reply:
I set it to a const for now. with pagination in the very near future I think it would make sense to set it to something like 1 second per nodeUser (that's being generous -- maybe more like 0.5 seconds per nodeUser)